client/src/components/feedback/comment.js

```diff
@@ -49,14 +98,51 @@ class FeedbackComment extends Component {
 
             <span className="answer" title="answer to feedback">answer to</span>
             <span className="mr-3 likeDislike">
-              <img src="/img/like.png" width="25px" height="25px" alt="like" title="like feedback" />
-              <small>{item.counterLike}</small>
+
+              {/* eslint-disable-next-line max-len */}
+              {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events,jsx-a11y/no-noninteractive-element-interactions */}
+              <img
+                src={likeImage}
+                width="25px"
+                height="25px"
+                alt="like"
+                title={title}
+                style={{ cursor: 'pointer' }}
+                onClick={onClick}
+              />
+              <small>{item.likes.length > 0 ? item.likes.length : ''}</small>
```

Comment:
> when like count is zero, there is no 'zero' shown for user. But when feedback is liked, user sees '1' in total likes count. This moves 'dislike' button to right side. It is needed to fix its position. Do somebody know how to implement this?

You can use 0 instead `''`

Reply:
Ok, but due to top web-apps, it is not good experience to show '0' value for user

Reply:
I'm gonna do this task after this one is done successfully. https://trello.com/c/AuVoxl8W/27-26as-an-authorized-user-of-the-system-i-want-to-have-a-possibility-to-dislike-feedback

Finally, it is two tasks in trello